src/index.js

```diff
@@ -4,6 +4,7 @@ import ReactDOM from 'react-dom';
 import { createStore, applyMiddleware, compose } from 'redux';
 import { Provider } from 'react-redux';
 import thunk from 'redux-thunk';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
There's a more react-elegant solution instead of Bootstrap.. In fact what would be good is to import only the modules we need to use, and there is a library which does that and has also quite nice visuals: http://www.material-ui.com/